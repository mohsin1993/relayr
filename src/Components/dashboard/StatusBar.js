import React from 'react';
import PropTypes from 'prop-types';

const StatusBar = ({ activeCount, inActiveCount }) => (
  <div className='status-bar'>
    <span>
      <label>Active: </label>
      <span className='value success'>{activeCount}</span>
    </span>
    <span>
      <label>Inactive: </label>
      <span className='value error'>{inActiveCount}</span>
    </span>
  </div>
);

StatusBar.propTypes = {
  activeCount: PropTypes.number,
  inActiveCount: PropTypes.number,
};

StatusBar.defaultProps = {
  activeCount: 0,
  inActiveCount: 0,
};

export default StatusBar;
