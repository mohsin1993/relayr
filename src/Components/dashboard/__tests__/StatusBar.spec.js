import React from 'react';
import snapshot from 'check-snapshot';
import StatusBar from '../StatusBar';

describe('StatusBar', () => {
  it('Render status bar correctly', () => {
    snapshot(<StatusBar activeCount={5} inActiveCount={2} />);
  });
});
