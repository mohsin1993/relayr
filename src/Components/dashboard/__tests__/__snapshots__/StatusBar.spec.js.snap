// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatusBar Render status bar correctly 1`] = `
<div
  className="status-bar"
>
  <span>
    <label>
      Active: 
    </label>
    <span
      className="value success"
    >
      5
    </span>
  </span>
  <span>
    <label>
      Inactive: 
    </label>
    <span
      className="value error"
    >
      2
    </span>
  </span>
</div>
`;
